### PR TITLE
Update validation scripts to validate for at least one base demon with zero decimals in native Tokens

### LINKED ISSUE #50: Update validation scripts to validate for at least one base demon with zero decimals in native Tokens
### Description
Update validation scripts to validate for at least one base demon with zero decimals in nativeTokens.json

### Motivation
Validation scripts needs to be updated to support atleast 1 base denom with zero decimals

### PR DESCRIPTION
**Pull request by vardan10**
### What was the problem?
This PR resolves #50 

### How was it solved?
- [x] Update validation scripts to validate for at least one base demon with zero decimals in native Tokens
- [x] Upgrade axios
- [x] Update fetching of certs from url

### How was it tested?
Local

### CODE REVIEW

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
The code changes work as expected. Please consider the following changes. Additionally, we might also need to update the README on how to extract the `apiCertificatePublicKey` from the URLs with an example.

File: utility/extractPublicKeyFromURL/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Prerequisites
+
+Before using the script, kindly ensure that you have Node.js installed on your system. You can download Node.js from the [official website](https://nodejs.org).
+
+## Getting Started
+
+1. **Download the script:**
+
+   Download the [`extractPublicKeyFromURL.js`](https://github.com/LiskHQ/app-registry/blob/main/utility/extractPublicKeyFromURL/extractPublicKeyFromURL.js) file from this repository and save it to a location on your computer.
```
Comment (sameersubudhi) [inline utility/extractPublicKeyFromURL/README.md]: Can we please hyperlink this script relative to the README instead?

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: utility/extractPublicKeyFromCertificate/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Before using the script, kindly ensure that you have Node.js installed on your s
 
 1. **Download the script:**
 
-   Download the [`extractPublicKeyFromCertificate.js`](https://github.com/LiskHQ/app-registry/blob/main/utility/extractPublicKeyFromCertificate.js) file from this repository and save it to a location on your computer.
+   Download the [`extractPublicKeyFromCertificate.js`](./extractPublicKeyFromCertificate.js) file from this repository and save it to a location on your computer.
```
Comment (nagdahimanshu) [inline utility/extractPublicKeyFromCertificate/README.md:13]: This file `./extractPublicKeyFromCertificate.js` has copyright section missing.

**Review by nagdahimanshu — APPROVED** [review]

**Pull request merged**